app/src/main/java/com/brandon/forzenbook/repository/ForzenRepositoryImpl.kt

```diff
@@ -14,103 +15,71 @@ class ForzenRepositoryImpl(
-                                                token = entityData.toLoginToken()
-                                                token
-                                            } catch (e: Exception) {
-                                                println("Brandon_Test_Data Failed to Convert Entity to Login Token. $e")
-                                                LoginToken(token = null, isDatabaseError = true)
-                                            }
-                                        } catch (e: Exception) {
-                                            println("Brandon_Test_Data Failed to Insert entity Data into Database. $e")
-                                            LoginToken(token = null, isDatabaseError = true)
-                                        }
-                                    } else {
-                                        println("Brandon_Test_Data Entity Data is Null After Conversion.")
-                                        return LoginToken(token = null, isDatabaseError = true)
-                                    }
-                                } catch (e: Exception) {
-                                    println("Brandon_Test_Data Failed to Convert Response to Entity. ${response.body()}")
-                                    return LoginToken(token = null, isServiceError = true)
-                                }
-                            } else {
-                                println("Brandon_Test_Data Unexpected Response Code ${response.code()}")
-                                return LoginToken(token = null, isServiceError = true)
+                            entityData = response.body()?.toForzenEntity()
+                            if (entityData != null) {
+                                forzenDao.insert(entityData)
+                                token = entityData.toLoginToken()
+                                return token
                             }
                         } catch (e: Exception) {
-                            println("Brandon_Test_Data Network Call Failed. $e")
-                            return LoginToken(token = null, isServiceError = true)
+                            Log.e("Brandon_Test_Data","Failed to Convert Response to Entity. ${response.body()}")
+                            return null
                         }
                     } else {
-                        println("Brandon_Test_Data Network No Internet Access.")
-                        return LoginToken(token = null, isNetworkError = true)
+                        Log.e("Brandon_Test_Data","Unexpected Response Code ${response.code()}")
+                        return null
                     }
                 } catch (e: Exception) {
-                    println("Brandon_Test_Data Error Deleting Data from Database $e")
-                    return LoginToken(token = null, isDatabaseError = true)
+                    Log.e("Brandon_Test_Data","Network Call Failed. $e")
```

Comment:
Remember constant for test tag everywhere

Reply:
added tags